### PR TITLE
conda env first steps

### PR DESCRIPTION
**Pull request by leafty**
Merge after https://github.com/SwissDataScienceCenter/renku-project-template/pull/9

### CODE REVIEW

File: docs/user/firststeps.rst [hunk 1]
```diff
@@ -156,34 +156,166 @@ Opening the file, we can see it contains some data in CSV format.
 Install and manage Python packages
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Continuing in the same terminal session as in last step, we can install python
-packages as usual with ``pip``:
+Here we will see how to use `conda <https://conda.io/docs/index.html>`_
```
Comment (rokroskar) [inline docs/user/firststeps.rst]: Maybe we should say conda is one way of managing environments and point to a doc somewhere explaining differences between virtual env and conda

File: docs/user/firststeps.rst [hunk 2]
```diff
@@ -156,34 +156,166 @@ Opening the file, we can see it contains some data in CSV format.
...
+
+.. code-block:: console
+
+    conda env list
+    # Expected output:
+    # # conda environments:
+    # #
+    # base                  *  /opt/conda
+    # renku                    /opt/conda/envs/renku
+    # test-conda-python        /opt/conda/envs/test-conda-python
```
Comment (rokroskar) [inline docs/user/firststeps.rst]: This won't be there, remove

File: docs/user/firststeps.rst [hunk 3]
```diff
@@ -156,34 +156,166 @@ Opening the file, we can see it contains some data in CSV format.
...
+
+The ``*`` denotes the current environment. As you can see, we are using the
+base ``conda`` environment.
+To activate our environment, run the following:
+
+.. code-block:: console
+
+    source activate test-conda-python
```
Comment (rokroskar) [inline docs/user/firststeps.rst]: Wrong name

File: docs/user/firststeps.rst [hunk 4]
```diff
@@ -156,34 +156,166 @@ Opening the file, we can see it contains some data in CSV format.
...
+    # atomicwrites-1.2.1   |   11 KB | ###################################################################################################################################################### | 100%
+    # py-1.6.0             |  136 KB | ###################################################################################################################################################### | 100%
+    # pytest-3.8.0         |  317 KB | ###################################################################################################################################################### | 100%
+    # Preparing transaction: done
+    # Verifying transaction: done
+    # Executing transaction: done
+    # #
+    # # To activate this environment, use
+    # #
+    # #     $ conda activate test-conda-python
```
Comment (rokroskar) [inline docs/user/firststeps.rst]: Wrong name

File: docs/user/firststeps.rst (hunk 2 above)
Comment (leafty) [reply docs/user/firststeps.rst]: `s/test-conda-python/tutorial-zhbikes`

**Comment by rokroskar** [comment]
I think we are not going to use conda env for the time being

**Pull request closed without merging**